src/ORM/Locator/LocatorAwareTrait.php

```diff
@@ -58,4 +67,40 @@ public function getTableLocator(): LocatorInterface
         /** @var \Cake\ORM\Locator\LocatorInterface */
         return $this->_tableLocator;
     }
+
+    /**
+     * Set the default table.
+     *
+     * If the default table is already set it will not be overwritten.
+     *
+     * @param string $name Class name.
+     * @return void
+     */
+    protected function setDefaultTable(string $name): void
+    {
+        if ($this->defaultTable === null) {
```

Comment:
I assume this is for controllers to mimic the default table matching controller name? If so, should we assert that `$defaultTable` is null since users shouldn't be re-configuring.

Reply:
Don't really need this method, I got rid of it.